sirius-valley/lerni-mobile
# [LER-38] Program detail screen

On this PR: 
.- Program detail screen finshed.
.- New StyledLine component created.
.- Several icons created.

Steps to test it: 
1. Go to explore.
2. Press on any program, it should redirect to the program detail screen.
